.gitignore

```diff
@@ -56,6 +57,7 @@ treeherder/webapp/static/
 #media files in deployment
 treeherder/webapp/media/builds4hanalysis/builds4h_analysis.json
 treeherder/webapp/media/builds4hanalysis/builds4h_report.txt
+treeherder/webapp/media/revision
```

Comment:
This could be treeherder/webapp/media/*. Files in media are generated by the application, so it's safe to put the content of this folder in the ignore list.